config/default/classicFullStack.yaml

```diff
@@ -0,0 +1,240 @@
+apiVersion: dynatrace.com/v1beta1
```

Comment:
Had to duplicate `assets/samples/classicFullStack/classicFullStack.yaml` because of kustomize not allowing references to a different directory (check "is not in or below" statement):
```
  ~/develop/dynatrace-operator/config/olm/openshift on   restructure-dynakube-samples ⇣1 !2 ?1 ····································································································  at  15:13:57
❯ kustomize build -o output.yaml
Error: accumulating resources: accumulation err='accumulating resources from '../default': '/home/alberto/develop/dynatrace-operator/config/olm/default' must resolve to a file': recursed accumulation of path '/home/alberto/develop/dynatrace-operator/config/olm/default': accumulating resources: accumulation err='accumulating resources from '../../../assets/samples/classicFullStack.yaml': security; file '/home/alberto/develop/dynatrace-operator/assets/samples/classicFullStack.yaml' is not in or below '/home/alberto/develop/dynatrace-operator/config/olm/default'': must build at directory: '/home/alberto/develop/dynatrace-operator/assets/samples/classicFullStack.yaml': file is not directory
```

Reply:
Removed in a65348502a441e054e248d9fcc969c02db0ad19c